mgmt2/rpc/config/JsonRpcConfig.h

```diff
@@ -0,0 +1,74 @@
+/**
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+#pragma once
+
+#include <string>
+#include <string_view>
+#include <variant>
+
+#include "yaml-cpp/yaml.h"
+
+#include "tscore/Diags.h"
+
+namespace rpc::config
+{
+template <class... Ts> struct overloaded : Ts... {
```

Comment:
Should look at moving this to "ts_meta.h"

Reply:
This actually needs to be removed, as I did  drop the `std::variant`. If I use it again, I'll move this to `ts_meta.h`